pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2729,6 +2729,11 @@
     },
     "runtime.win10-arm64.runtime.native.System.IO.Compression": {
       "BaselineVersion": "4.4.0"
+    },
+    "System.Net.ServicePoint": {
+      "AssemblyVersionInPackageVersion": {
+        "4.0.0.0": "4.4.0"
```

Comment:
@ericstj @weshaggard Since we're adding a new contract, do we want to match the binary version with the package version?